### PR TITLE
bump model to version 0.0.101

### PR DESCRIPTION
**Pull request by ciaranRoche**
Closes JIRA : https://issues.redhat.com/browse/SDA-3255

### CODE REVIEW

**Comment by ciaranRoche** [comment]
\cc @vkareh PTAL

**Comment by vkareh** [comment]
@ciaranRoche this LGTM, can you add a second commit with the updates to `CHANGES.md` and a new version?

File: CHANGES.adoc [hunk 1]
```diff
@@ -3,6 +3,9 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+== 0.1.151 Dec 2 2020
+- update model to 0.0.101
```
Comment (vkareh) [inline CHANGES.adoc:7]: There are a bunch more changes to add here :)
```
~/g/s/g/o/ocm-sdk-go (⎇ master) $ git log v0.1.150..HEAD --oneline --no-merges --no-decorate --reverse 
0b51bf9 Move logging code to `logging` package
9733ca7 Rename `Metrics` to `MetricsSubsystem`
efcfb1b Add method to read metrics subsystem
0903d9f Load metrics subsystem configuration from file
c881fc8 Load string if it doesn't look like a file
eec027b Load `os.Stdin` in dump configuration example
08d16f0 Reject URLs without scheme or host name
eaa2d83 Add redirection tests
```

Comment (ciaranRoche) [reply CHANGES.adoc:7]: damn, my bad, ill add them now

**Comment by vkareh** [comment]
LGTM

**Pull request merged**